src/platform/site-wide/sass/overrides/uswds/focus.scss

```diff
@@ -0,0 +1,138 @@
+@mixin focus-gold-light-outline {
+  outline: 2px solid $color-gold-light;
+  outline-offset: 2px;
+}
+
+@mixin focus-gold-lighter-highlight {
+  background: $color-gold-lighter;
+  outline: 2px solid $color-gold-lighter;
+  outline-offset: 0;
+}
+
+a:focus {
```

Comment:
The rules for a, input, button, and select feel like they should be in the design system.